packages/@aws-cdk/aws-batch-alpha/test/ecs-job-definition.test.ts

```diff
@@ -74,3 +74,45 @@ test('can be imported from ARN', () => {
   expect(importedJob.jobDefinitionArn).toEqual('arn:aws:batch:us-east-1:123456789012:job-definition/job-def-name:1');
```

Comment:
Can you add a second test that does the same thing but uses an imported definition. This will make it easier to validate that the return value is correct.